ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -104,6 +115,7 @@ class ( SL.ShelleyBasedEra era
       , HasField "_nOpt" (Core.PParams era) Natural
       , HasField "_rho" (Core.PParams era) SL.UnitInterval
       , HasField "_tau" (Core.PParams era) SL.UnitInterval
+
```

Comment:
new line?

Reply:
Ah yes. I was indecisive here. This was stale. Thanks.

Fixed!